timemachine/optimize/protocol.py

```diff
@@ -208,6 +219,56 @@ def max_work_stddev_distance(prev_lam, next_lam, max_step=0.25):
     return max_work_stddev_distance
 
 
+# (2) pair overlap
+def reweighted_pair_overlap(u_n_A, u_n_B, u_n_ref):
+    """given arrays [u(x) for x in xs] (for u in {u_A, u_B, u_ref}),
+    estimate overlap(A, B) by reweighting from ref->A and ref->B"""
```

Comment:
Could we add a mathematical definition of the pair overlap to the docstring here, or point to where it's defined? I often am confused about the precise definition, whether or not there's a factor of 2, etc.

Reply:
Have added a literature reference here, and posted additional notes in an internal system.

Specifically about the factor of 2, I have been inconsistent about writing `p_a * p_b / (p_a + p_b)` (and then multiplying by an ad-hoc-looking `2`) vs. `p_a * p_b / (0.5 * (p_a + p_b))` (denominator is an equally weighted mixture of states `a` and `b`).

Reply:
Thanks. I didn't see this before I posted https://github.com/proteneer/timemachine/pull/1387#discussion_r1809225572; feel free to ignore that.

Reply:
I think the suggestion is cleaner -- applied in https://github.com/proteneer/timemachine/pull/1387/commits/1fb110a82c33093dd1f529660ef01d426c7fc416